virttest/qemu_vm.py

```diff
@@ -3018,7 +3021,7 @@ def _cleanup(self, free_mac_addresses):
 
         for f in file_list:
             try:
-                if f:
+                if f and isinstance(f, str):
```

Comment:
It looks this line should be 
```
if os.path.exists（f):
      os.unlink(f)
```

Reply:
@xutian Em, this update actually is not needed anymore, cause previously I made `f` a tuple instead of a string, something like `(127.0.0.1, 6001)`, `os.path.exists(f)` might also fail this way, I will just remove this update. Thanks.